.github/workflows/release-t0rn.yml

```diff
@@ -0,0 +1,262 @@
+name: t0rn Rococo Release
+
+# Trigger only on tags starting with test
+on:
+  push:
+    tags:
+      - "test[0-9]+"
+
+# TODO: uncomment after tests
+# on:
+#   push:
+#     tags:
+#       - "v[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+"
+
+env:
+  PARACHAIN_NAME: t0rn
+  RUST_BACKTRACE: full
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build-release:
+    runs-on: ["self-hosted", "rust"]
+    steps:
+      - name: ☁ Checkout git repo
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          token: ${{ secrets.GH_PAT }}
+
+      - name: ⚙️ Get nightly rust toolchain with wasm target
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly-2022-06-16
+          target: wasm32-unknown-unknown
+          components: rustfmt, clippy
+          override: true
+
+      - name: 🏭 Build circuit
+        uses: actions-rs/cargo@v1
+        continue-on-error: false
+        with:
+          command: build
+          args: --manifest-path ./node/${{ env.PARACHAIN_NAME }}-parachain/Cargo.toml --locked --release
+
+      - name: 📤 Upload collator binary
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ github.sha }}-${{ env.PARACHAIN_NAME }}-collator-release-artifacts
+          path: ./target/release/${{ env.PARACHAIN_NAME }}-collator
+          if-no-files-found: error
+          retention-days: 1
+
+      - name: 🫧 Build runtime WASM
+        run: |
+          ./scripts/build_wasm.sh ${{env.PARACHAIN_NAME}}
+
+      - name: 📤 Upload runtime WASM binary
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ github.sha }}-${{ env.PARACHAIN_NAME }}_parachain_runtime.compact.compressed.wasm
+          path: ./target/release/wbuild/${{ env.PARACHAIN_NAME }}-parachain-runtime/${{ env.PARACHAIN_NAME }}_parachain_runtime.compact.compressed.wasm
+          if-no-files-found: error
+          retention-days: 1
+
+      - name: 📤 Upload runtime WASM hash
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ github.sha }}-${{ env.PARACHAIN_NAME }}_parachain_runtime.compact.compressed.wasm.hash
+          path: ./target/release/wbuild/${{ env.PARACHAIN_NAME }}-parachain-runtime/${{ env.PARACHAIN_NAME }}_parachain_runtime.compact.compressed.wasm.hash
```

Comment:
Maybe instead of `.hash` lets suffix with the concrete hash algorithm used - i had been stuck on that b4 :)